tests/testthat/test-geom-violin.R

```diff
@@ -90,6 +98,10 @@ test_that("geom_violin draws correctly", {
   expect_doppelganger("quantiles",
     ggplot(dat, aes(x=x, y=y)) + geom_violin(draw_quantiles=c(0.25,0.5,0.75))
   )
+  dat_outlier <- data.frame(x = 1, y = c(0, 0.25, 0.5, 0.75, 5))
```

Comment:
Why do we need a visual test here?

Reply:
Good question - maybe we don't. The change in 
https://github.com/tidyverse/ggplot2/pull/4654/files/09b4df9ed1b61d1071c00b1d5a886f72d5a4e57d#diff-6ac793ea0483ba5c6cd55aa4b47e2a9fca57dc1d867fb0ee8851058dfae407d5R200
does not affect any of the existing visual tests, so I thought it might make sense to add a visual test that is affected by the fix and, in fact, make sure the (originally invisible) quantile line stays invisible.

Reply:
I replaced the visual test by one based on the `grob`. I haven't found good documentation on how to best traverse this tree, though, but it's a start.